### DISCUSSION TITLE
Working together with original Windows update

### DISCUSSION THREAD

**Discussion by wyf180** [LegacyUpdate/LegacyUpdate#32] [Questions & Help]
After instead LegacyUpdate,I still want to try original Windows update.I opened the ProxHTTPSProxy and the icon in the backup folder ,however the link still went to LegacyUpdate.net and can't get the original wu address .So could you fix the conflict ?

**Comment by kirb** [comment]
You’ll need to delete `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\WindowsUpdate\URL`, this overrides the url wupdmgr.exe (the Windows Update shortcut) opens to. You can also directly go to update.microsoft.com in IE.

**Comment by wyf180** [comment]
> You’ll need to delete `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\WindowsUpdate\URL`, this overrides the url wupdmgr.exe (the Windows Update shortcut) opens to. You can also directly go to update.microsoft.com in IE.

The website could open after edit rigedit but can't work with error 0x80096010.
